src/components/NcRichContenteditable/NcRichContenteditable.vue

```diff
@@ -1001,6 +1015,23 @@ export default {
 				this.getTributeContainer().classList.remove(this.$style['tribute-container--focus-visible'])
 			}
 		},
+
+		showTributeManually(trigger) {
+			// Programmatic show of the tribute menu
+			this.$refs.contenteditable.focus()
+			let index = 0 // default to first collection '@'
+			if (trigger === ':') {
+				index += this.linkAutocomplete
+			} else if (trigger === '/') {
+				index += this.linkAutocomplete + this.emojiAutocomplete
+			}
+			this.tribute.showMenuForCollection(this.$refs.contenteditable, index)
+		},
+
+		hideTributeManully() {
```

Comment:
There is no alternative for non-manual hiding, calling a function is always a manual call 😀

```suggestion
		hideTribute() {
```